elements/lisk-validator/src/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { ErrorObject } from 'ajv';
+
+export { DataValidateFunction, DataValidationCxt } from 'ajv/dist/types';
```

Comment:
Prefer not to introduce types from non exported files

Reply:
Yes we should not, but that's what they described in their docs. 

https://ajv.js.org/json-type-definition.html

Reply:
ah ok! then it should be fine. They will probably manage the breaking change properly if it's in the doc